prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -110,10 +110,10 @@ do {                                                                	\
 	 PEND_SEND_IS_LIMITED(_ep))
 
 struct fi_ibv_rdm_header {
-	uint64_t imm_data;          // TODO: not implemented
+/*	uint64_t imm_data; TODO: not implemented */
```

Comment:
No, this field was reserved for implementing of OFI tagged imm_data. It's not supported yet.